config/settings/base.py

```diff
@@ -29,18 +29,12 @@
 # though not all of them may be available with every OS.
 # In Windows, this must be set to your system time zone.
 TIME_ZONE = "UTC"
-# https://docs.djangoproject.com/en/dev/ref/settings/#language-code
-LANGUAGE_CODE = "en-gb"
```

Comment:
Is this actually no longer needed? Probably not, just doesn't seem directly related to I18N.

Reply:
Looking at the docs, in django it's only used for localisation. I couldn't see any more references of it either so looks safe to remove